torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -33,22 +33,21 @@ def _orthogonalize(matrix, epsilon=1e-8):
 
 
 class PowerSGDState(object):
-    """
+    r"""
     Stores both the gradient compression configs and the internal states for all the gradients during the training.
-    Particularly, `matrix_approximation_rank` and `start_powerSGD_iter` are the main configs that need to be tuned by the user.
-    Although `use_error_feedback` and `warm_start` can also be tuned by the user,
+    Particularly, ``matrix_approximation_rank`` and ``start_powerSGD_iter`` are the main configs that need to be tuned by the user.
+    Although ``use_error_feedback`` and ``warm_start`` can also be tuned by the user,
     they are typically turned on for performance.
 
-    Note [Guidance to Tune `matrix_approximation_rank` And `start_powerSGD_iter`]
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~
-    1) To tune `matrix_approximation_rank`, the user can increase it from 1 by factors of 2,
+    1. To tune ``matrix_approximation_rank``, the user can increase it from 1 by factors of 2,
     until a satisfying accuracy can be reached.
-    The increase of `matrix_approximation_rank` can substantially increase the computation costs of the compression.
-    However, the accuracy may not be futher improved beyond a certain `matrix_approximation_rank` value.
-    2) To tune `start_powerSGD_iter`, the user can typically start with 10% of total training steps,
+    The increase of ``matrix_approximation_rank`` can substantially increase the computation costs of the compression.
+    However, the accuracy may not be futher improved beyond a certain ``matrix_approximation_rank`` value.
+
+    2. To tune ``start_powerSGD_iter``, the user can typically start with 10% of total training steps,
     and increase it until a satisfying accuracy can be reached.
     Deferrring PowerSGD can effectively improve the accuracy,
-    even a relatively small `matrix_approximation_rank` is used.
+    even a relatively small ``matrix_approximation_rank`` is used.
     This is because that, the beginning of training phase is usually very sensitive to inaccurate gradients,
     and compressing gradients too early may make the training quickly take a suboptimal trajectory,
     which can result in an irrecoverable impact on the accuracy.
```

Comment:
Suggested rephrasing:
```
Stores both the algorithm's hyperparameters and the internal state for all the gradients during the training.

The settings ``matrix_approximation_rank`` and ``start_powerSGD_iter`` should be tuned by the user. 
- To tune ``matrix_approximation_rank``, start from 1 and increase by factors of 2 until a satisfactory accuracy is reached. 
A too low ``matrix_approximation_rank`` can yield loss in accuracy, while a high ``matrix_approximation_rank`` yields less communication savings and slower compression. 
- To tune ``start_powerSGD_iter`, we suggest to start with 10% of the total training steps, and increasing it until satisfactory accuracy is reached. Because the beginning of training is the most sensitive to accurate updates, starting to compress communication only after some iterations improves accuracy, even when a relatively small ``matrix_approximation_rank`` is used.

We suggest to keep the binary hyperparameters ``use_error_feedback`` and ``warm_start`` on.
```